libs/langchain/tests/integration_tests/chains/test_graph_database.py

```diff
@@ -252,3 +263,122 @@ def test_cypher_save_load() -> None:
     qa_loaded = load_chain(FILE_PATH, graph=graph)
 
     assert qa_loaded == chain
+
+
+def test_exclude_types() -> None:
+    """Test exclude types from schema."""
+    url = os.environ.get("NEO4J_URL")
+    username = os.environ.get("NEO4J_USERNAME")
+    password = os.environ.get("NEO4J_PASSWORD")
+    assert url is not None
+    assert username is not None
+    assert password is not None
+
+    graph = Neo4jGraph(
+        url=url,
+        username=username,
+        password=password,
+    )
+    # Delete all nodes in the graph
+    graph.query("MATCH (n) DETACH DELETE n")
+    # Create two nodes and a relationship
+    graph.query(
+        "CREATE (a:Actor {name:'Bruce Willis'})"
+        "-[:ACTED_IN]->(:Movie {title: 'Pulp Fiction'})"
+        "<-[:DIRECTED]-(p:Person {name:'John'})"
+    )
+    # Refresh schema information
+    graph.refresh_schema()
+
+    chain = GraphCypherQAChain.from_llm(
+        OpenAI(temperature=0), graph=graph, exclude_types=["Person", "DIRECTED"]
+    )
+    expected_schema = (
+        "Node properties are the following: \n"
+        " {'Movie': [{'property': 'title', 'type': 'STRING'}], "
```

Comment:
Not for now but for the future should we have all the extra symbols [], {} in the prompt for the LLM or would it be better to have something like a plain text representation?
 e.g. 

```
* Movie has title:String
* Actor has name: String, dob: date
```

Reply:
Anecdotal and not rigorously tested, but when I was iterating on the format for the graph schema used to inform the LLM, I found that the more "human readable" the text was the better the Cypher prompt generation became. Still lots to explore and optimize in schema format.